src/index.js

```diff
@@ -756,11 +763,20 @@ async function start() {
         if (hubState == null) {
           return;
         }
+
+        let author_name = msg.guild.members.cache.get(msg.author.id).nickname;
```

Comment:
Would there be value in using the Optional Chaining operator here [?.] in case msg, guild, members or cache is null?

Reply:
I don't think we need to worry about msg, but the guild, members, cache, and the result of the get could probably benefit from it.  Thanks.  Updated.